src/PrestaShopBundle/Form/Admin/Type/DatePickerType.php

```diff
@@ -37,6 +39,16 @@
  */
 class DatePickerType extends AbstractType
 {
+    /**
+     * @var ArabicToLatinDigitDataTransformer
+     */
+    private $arabicToLatinDigitDataTransformer;
+
+    public function __construct(ArabicToLatinDigitDataTransformer $arabicToLatinDigitDataTransformer)
```

Comment:
Shouldn't this be a DataTransformerInterface?

```suggestion
    public function __construct(DataTransformerInterface $arabicToLatinDigitDataTransformer)
```